Web-Programming/OpenWeatherMaps.js

```diff
@@ -0,0 +1,31 @@
+const fetch = require('node-fetch')
```

Comment:
what about this requirement, 
we can add a common package.json file which can be used to install all requirements using
`npm install`

and please remove **package-lock.json** file
thank you @ruppysuppy

Reply:
and please add a global package.json file

Reply:
I have added a global package.json file and removed package-lock.json but the HeapSort was causing some issues, I didn't touch it but it was modified by itself.